tests/core/test_stellargraph.py

```diff
@@ -19,6 +19,7 @@
 import networkx as nx
 from stellargraph.core.graph import *
 from stellargraph.data.converter import *
+from ..test_utils.alloc import snapshot, allocation_benchmark
```

Comment:
Copyright (earlier in file - can't seem to make suggestions for unmodified lines) should become `2017-2019` as this file has been touched.

Reply:
I've updated it, but this seems like somewhat distracting busy-work for something of limited value, especially since these aren't the first edits this year (although they are the first non-trivial ones):

![image](https://user-images.githubusercontent.com/1203825/69688772-1dcd3b80-111b-11ea-8459-384b53465994.png)

Reply:
I agree, others clearly failed to follow the procedure. But since it is a publicly visible library, it is part of the standard checks, like blacking the code and running the unit tests. Might as well fix these things when we notice them, trivial though they are.

Reply:
I've filed an issue https://github.com/stellargraph/stellargraph/issues/533 and opened a pull request #534, so that there's no need for a human procedure here and it's enforced automatically 😄

Reply:
> no need for a human procedure here

Correct me if I'm wrong, but I thought CI could check but not fix this automatically (due to `mtime` issues), so people still need to manually adjust the year?

Reply:
Yes, but no one will have to remember to think about it: it'll be automatically flagged and so remind PR authors to fix it, and reviewers can just rely on that CI action instead of having to catch it themselves.

Reply:
Yeah, but for me it is on the order of forgetting to `black` the code and thus having to do a second commit - yes, great that CI catches it, but should have been on a mental checklist before pushing the commits. And I'm not trying to disparage your effort, I think it is a great new check.